docs/administration/information_schema.md

```diff
@@ -82,7 +82,7 @@ The following fields are provided in `tables_config`:
 | TABLE_SCHEMA     | Name of the database that stores the table.                  |
 | TABLE_NAME       | Name of the table.                                           |
 | TABLE_ENGINE     | Engine type of the table.                                    |
-| TABLE_MODEL      | Data model of the table. Valid values: "DUP_KEYS", "AGG_KEYS", "UNQ_KEYS" or "PRI_KEYS". |
+| TABLE_MODEL      | Table type. Valid values: "DUP_KEYS", "AGG_KEYS", "UNQ_KEYS" or "PRI_KEYS". |
```

Comment:
![image](https://user-images.githubusercontent.com/57167462/233816768-2311bcfd-f3db-4a87-a1d9-f59327909fee.png)
The table type of the table?